resolwe/flow/managers/listener/python_process_plugin.py

```diff
@@ -481,14 +482,28 @@ def handle_create_object(
 
         :raises RuntimeError: if user has no permission to create the object.
         """
+
+        @retry(
+            max_retries=10,
```

Comment:
Retries are hard coded here. Would it make sense to fetch this information from `resolwe.flow.models.base MAX_SLUG_RETRIES`?